src/main/java/StressMain.java

```diff
@@ -690,6 +749,19 @@ private static CloudSolrClient buildSolrClient(SolrCloud cloud) {
 		return new CloudSolrClient.Builder().withZkHost(cloud.getZookeeperUrl()).withZkChroot(cloud.getZookeeperChroot()).build();
 	}
 
+	/**
+	 * buildHttpSolrClient tries to build the client pointing to the overseer node, otherwise
+	 * it returns a client pointing to the first node in cloud.nodes
+	 */
+	private static HttpSolrClient buildHttpSolrClient(SolrCloud cloud, String overseerLeader) {
```

Comment:
Nice to have this!! Should we name overseerLeader even more generic such as preferredNodeName? I'm thinking this could be useful for other usages as well (preferred node to send query to or index data to?)